packages/flutter_driver/test/src/real_tests/find_test.dart

```diff
@@ -70,10 +71,14 @@ void main() {
       'firstMatchOnly': 'true',
     };
 
-    final Descendant a = Descendant.deserialize(serialized);
+    final Descendant a = Descendant.deserialize(serialized, TestDeserialize());
     expect(a.of, isA<ByType>());
     expect(a.matching, isA<ByValueKey>());
     expect(a.matchRoot, isTrue);
     expect(a.firstMatchOnly, isTrue);
   });
 }
+
+class TestDeserialize with DeserializeFinderFactory{
+
+}
```

Comment:
We should probably write a test that actually implements new logic here and makes sure it works.

Also, please add a new line at EOF.

Reply:
```suggestion
class TestDeserialize with DeserializeFinderFactory{

}

```

Reply:
I mixed it with Mock by referring to other codes